pymilvus/client/async_grpc_handler.py

```diff
@@ -0,0 +1,705 @@
+                    self._address,
+                    options=opts,
+                )
+            else:
+                if self._server_name != "":
+                    opts.append(("grpc.ssl_target_name_override", self._server_name))
+
+                root_cert, private_k, cert_chain = None, None, None
+                if self._server_pem_path != "":
+                    with Path(self._server_pem_path).open("rb") as f:
+                        root_cert = f.read()
+                elif (
+                    self._client_pem_path != ""
+                    and self._client_key_path != ""
+                    and self._ca_pem_path != ""
+                ):
+                    with Path(self._ca_pem_path).open("rb") as f:
+                        root_cert = f.read()
+                    with Path(self._client_key_path).open("rb") as f:
+                        private_k = f.read()
+                    with Path(self._client_pem_path).open("rb") as f:
+                        cert_chain = f.read()
+
+                creds = grpc.ssl_channel_credentials(
+                    root_certificates=root_cert,
+                    private_key=private_k,
+                    certificate_chain=cert_chain,
+                )
+                self._async_channel = grpc.aio.secure_channel(
+                    self._address,
+                    creds,
+                    options=opts,
+                )
+
+        # avoid to add duplicate headers.
+        self._final_channel = self._async_channel
+        if self._log_level:
+
+            async_log_level_interceptor = async_header_adder_interceptor(
+                ["log_level"], [self._log_level]
+            )
+            self._final_channel._unary_unary_interceptors.append(async_log_level_interceptor)
+
+            self._log_level = None
+        if self._request_id:
+
+            async_request_id_interceptor = async_header_adder_interceptor(
+                ["client_request_id"], [self._request_id]
+            )
+            self._final_channel._unary_unary_interceptors.append(async_request_id_interceptor)
+
+            self._request_id = None
+        self._async_stub = milvus_pb2_grpc.MilvusServiceStub(self._final_channel)
+
+    def _setup_identifier_interceptor(self, user: str, timeout: int = 10):
+        host = socket.gethostname()
+        self._identifier = self.__async_internal_register(user, host, timeout=timeout)
+        _async_identifier_interceptor = async_header_adder_interceptor(
+            ["identifier"], [str(self._identifier)]
+        )
+        self._async_channel._unary_unary_interceptors.append(_async_identifier_interceptor)
+        self._async_stub = milvus_pb2_grpc.MilvusServiceStub(self._async_channel)
+
+    @property
+    def server_address(self):
+        return self._address
+
+    def get_server_type(self):
+        return get_server_type(self.server_address.split(":")[0])
+
+    @retry_on_rpc_failure()
+    async def async_create_collection(
```

Comment:
Please rename `async_create_collection` to `create_collection`. Since we no longer need AsyncGrpcHandler to deal with sync tasks, and `async def` already indicating this is an async method,  it's better not to "repeat yourself" when naming a function.

Ditto from all the following `async_*` methods